.github/workflows/ci-prq.yml

```diff
@@ -24,7 +24,8 @@ jobs:
       - name: Build with Gradle
         env:
           CI: true
-        run: ./gradlew --parallel clean quality
+        run: |
+            sudo env "PATH=$PATH" bash -c "ulimit -l 65536 && ulimit -a && ./gradlew --no-daemon --parallel clean quality"
```

Comment:
can you apply similar changes as done in `ci-prb.yml`?

Reply:
I added 'gradle-command' to ci-prb.yml

Also, I modified the Gradle command string in `ci-snapshot.yml` and `ci-release.yml`

Reply:
I have remove 'gradle-command' based on feedback from Idel.